### ISSUE TITLE
Capture page view frequency

### ISSUE THREAD

**Issue by lauraGgit** [18F/fs-open-forest-platform#470]
## Notes
When running over some of the reports, we realized that not all the page views were being captured (as our pre-launch tests has indicated) which throws off some of the visualizations we created. 
If we know which pages people are going on, we could improve their customer experience.

## Acceptance Criteria
- [x] In Google analytics, I can see subsequent pages that a user visited beyond the Open Forest landing page.
- [x] We can see how much the user spent on each subsequent page.

## Tasks
- [x] add ga.pageview events to individual forest pages/ buy permit pages. (the permit confirmation page and landing pages are mostly covered)
- [x] Once in staging verify that page views are being sent.
- [x] PO approved.

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] Server actions captured by logs
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by lauraGgit** [comment]
Modelling it off of this: https://codeburst.io/using-google-analytics-with-angular-25c93bffaa18

**Comment by ASprinkle** [comment]
from stand up 11/30: Mike will follow up with Laura

**Comment by lauraGgit** [comment]
@mtlaney - I pushed up and implementation based on [this](https://stackoverflow.com/a/46463247), but still getting a circle error suggesting its not picking up the `types` thoughts? https://circleci.com/gh/18F/fs-open-forest-platform/3306

**Comment by mtlaney** [comment]
@lauraGgit checking it out now.

**Comment by mtlaney** [comment]
still failing unit tests, continuing to look into.

**Comment by lauraGgit** [comment]
@davemcorwin looks like the pr had an e2e failure, rebuilding to see if that fixed it: https://circleci.com/gh/18F/fs-open-forest-platform/3420

**Comment by lauraGgit** [comment]
Ok, it failed again- based off of an e2e test in login.gov -> in the authenticated tests

**Comment by ASprinkle** [comment]
from stand up: submitted a new PR, waiting on peer review.

**Comment by ASprinkle** [comment]
Mike submitted PR under #513

**Comment by ASprinkle** [comment]
@davemcorwin will confirm merge status

**Comment by ASprinkle** [comment]
@davemcorwin checking in on the acceptance criteria: Back in Dec I had some road blocks setting up an account for access to view google analytics associated with Open Forest. Can you confirm that the two A/C above are reflected, and if so, check the boxes.  If possible, a screen shot would be helpful too. Thank you!

**Comment by ASprinkle** [comment]
From stand up: Verified that it is not collecting internal page views.

**Comment by ASprinkle** [comment]
This is great progress!  What does page view frequency look like in the DAP report?

**Comment by davemcorwin** [comment]
@ASprinkle once ☁️ .gov is back, the deployment to staging can finish and i'll let you know!

**Comment by davemcorwin** [comment]
Looks to be working on staging, see the network requests to the `collect` endpoint below, will confirm in google analytics as well:

![screen shot 2019-02-19 at 9 16 23 am](https://user-images.githubusercontent.com/6662371/53034222-9a621980-3427-11e9-8b53-afeb2d97c578.png)

**Comment by davemcorwin** [comment]
Confirmed all pageviews are available in Staging. In particular, the mt hood pageview was from in ap navigation.

![screen shot 2019-02-19 at 10 57 30 am](https://user-images.githubusercontent.com/6662371/53039993-2f1f4400-3435-11e9-9a54-4d13b8ba6184.png)
Google Analytics

**Comment by ASprinkle** [comment]
Way to go!!
